code/datums/ai_laws/ai_laws.dm

```diff
@@ -79,8 +79,14 @@ GLOBAL_VAR(round_default_lawset)
 /proc/pick_weighted_lawset()
 	var/datum/ai_laws/lawtype
 	var/list/law_weights = CONFIG_GET(keyed_list/law_weight)
+	var/list/specified_law_ids = CONFIG_GET(keyed_list/specified_laws) // BUBBER EDIT
+
```

Comment:
```suggestion
	var/list/specified_law_ids = CONFIG_GET(keyed_list/specified_laws) // BUBBER EDIT
```

Reply:
no file diff here?

Reply:
I was just going to have the stray line of empty space removed.